unity/Holo/Assets/Scripts/ModelClippingPlaneControl.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using HoloToolkit.Unity.Buttons;
+using HoloToolkit.Unity.InputModule;
+using HoloToolkit.Unity.UX;
+
+public class ModelClippingPlaneControl : MonoBehaviour, IClickHandler
+{
+    public CompoundButton ClippingPlaneBtn;
+    public CompoundButton ClippingPlaneTranslationBtn;
+    public CompoundButton ClippingPlaneRotationBtn;

```

Comment:
I would suggest to name these fields `ButtonXxx` instead of `XxxBtn`, to make them consistent with similar names in https://github.com/MicroscopeIT/holo/blob/master/unity/Holo/Assets/Scripts/model_with_plate/ModelWithPlate.cs#L20 .

Reply:
done